cmake/modules/FindPython.cmake

```diff
@@ -29,11 +29,12 @@ else()
   list(APPEND PYTHON_LIBRARIES ${PC_PYTHON_STATIC_LIBRARIES})
 endif()
 
+list(APPEND PYTHON_DEFINITIONS -DHAS_PYTHON=1)
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(PYTHON REQUIRED_VARS PYTHON_INCLUDE_DIR PYTHON_LIBRARY PYTHON_LIBRARIES)
+find_package_handle_standard_args(PYTHON REQUIRED_VARS PYTHON_INCLUDE_DIR PYTHON_LIBRARY PYTHON_LIBRARIES PYTHON_DEFINITIONS)
```

Comment:
is there a reason why you add it here? Usually we only add vars that are the result of find_* (and that have a chance to be set to NOT-FOUND)